src/main/java/net/earthcomputer/clientcommands/command/TranslateCommand.java

```diff
@@ -0,0 +1,66 @@
+package net.earthcomputer.clientcommands.command;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonParser;
+import com.mojang.brigadier.Command;
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import com.mojang.brigadier.exceptions.SimpleCommandExceptionType;
+import net.earthcomputer.clientcommands.command.arguments.TranslationQueryArgumentType;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.text.Text;
+import org.apache.http.client.utils.URIBuilder;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.charset.Charset;
+import java.time.Duration;
+
+import static net.earthcomputer.clientcommands.command.arguments.TranslationQueryArgumentType.*;
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.*;
+
+public class TranslateCommand {
+
+    private static final SimpleCommandExceptionType UNKNOWN_ERROR_EXCEPTION = new SimpleCommandExceptionType(Text.translatable("commands.ctranslate.unknownError"));
+
+    private static final String url = "https://translate.googleapis.com/translate_a/single?client=gtx&dt=t";
+
+    private static final HttpClient httpClient = HttpClient.newHttpClient();
+
+    private static final Duration DURATION = Duration.ofSeconds(5);
```

Comment:
Would it make sense to move these fields to a separate class to reduce duplication with the `PlayerInfoCommand` class?

Reply:
I guess? I'll let @Earthcomputer decide.

Reply:
I think we should do this when at least three classes use these fields.